sdk/python/api/hopr.py

```diff
@@ -570,6 +570,7 @@ async def check_url():
                 while not ready:
                     try:
                         ready = (await s.get(url, timeout=0.3)).status == 200
+                        await asyncio.sleep(0.5)
```

Comment:
Why is this needed, when it wasn't before?

Reply:
Because the while loop is hammering the API each microsecond like crazy, I want to spare the fans on my non-ARM machines